libvast/src/system/configuration.cpp

```diff
@@ -103,16 +103,16 @@ caf::error configuration::parse(int argc, char** argv) {
     std::error_code err{};
     const auto exists_conf_yaml = std::filesystem::exists(conf_yaml, err);
     const auto exists_conf_yml = std::filesystem::exists(conf_yml, err);
-    if (err)
-      return caf::make_error(ec::filesystem_error,
-                             fmt::format("failed to check if vast yaml config "
-                                         "files existed from directory {}: {}",
-                                         dir, err.message()));
+    if (err) {
+      VAST_VERBOSE("failed to check if vast.yaml file exists in {}: {}", dir,
+                   err.message());
```

Comment:
I would assume that this is normally not intended.
```suggestion
      VAST_WARN("failed to check if vast.yaml file exists in {}: {}", dir,
                err.message());
```

Reply:
I intentionally used verbose over warn because this can happen in sandboxed environments and is not usually worth warning about. @0snap what do you think?

Reply:
What if `/etc/vast` is accidentally not readable for the `vast` user?

Reply:
Fair point. Will change, then merge after CI is done.